app/views/card/policy_with_member_card.html.haml

```diff
@@ -0,0 +1,33 @@
+.card-container
+    %p.logo 
+        %span.subtext They Vote For You 
+    .row
+        .col-sm-12 
+            .image-card 
+                .member-image 
+                    %img{:src => @member.large_image_url} 
```

Comment:
You can use the more modern ruby 1.9 syntax here `%img{src: @member.large_image_url}`